go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/nuclio/errors v0.0.1
 	github.com/nuclio/logger v0.0.1
 	github.com/nuclio/zap v0.0.2
+	github.com/pavius/impi v0.0.3 // indirect
```

Comment:
I think this shouldn't be here (go mod tidy)

Reply:
it should, it's used when linting
i pushed go mod tidy commit

Reply:
Doesn't mean it should stay there.